src/lib/GraphQLUtils/validator.ts

```diff
@@ -0,0 +1,22 @@
+import { Validator } from 'types/validator';
+import { DesmosProfile } from 'types/desmos';
+
+/**
+ * Format an incoming validator data from the server into a format that is easier to parse by the app.
+ * @param validator - Validator data retrieved from the server.
+ * @param profile - Optional validator profile.
+ * @return {Validator} - A formatted Validator object.
+ */
+export function convertGraphQLValidator(validator: any, profile?: DesmosProfile): Validator {
+  return {
+    operatorAddress: validator.validator_info.operator_address,
+    selfDelegateAddress: validator.validator_info.self_delegate_address,
+    commission: validator.validator_commissions[0]?.commission ?? 1,
```

Comment:
This can still throw an error: if `validator.validator_commissions` is an empty array, `validator.validator_commissions[0]` will throw an `ArrayIndexOutOfBoundsException`

Reply:
no, if you get the first element of an empty arry will return `undefined`

Reply:
So that array will always be defined? It's not possible for it to be `undefined`?